website_docs/manual.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Instrumentation
```

Comment:
Should the full title be **Manual instrumentation**?
If so, then after making that change, consider adding `linkTitle: Manual`